### DISCUSSION TITLE
Hook for removing the "view cart" button for the mini cart isn't working

### DISCUSSION THREAD

**Discussion by finalwebsites** [bootscore/bootscore#798] [Q&A]
Hi,

inside woocommerce/cart/mini-cart.php there is the action called "woocommerce_widget_shopping_cart_buttons". 
So I tried to remove the "View cart" using 
`remove_action( 'woocommerce_widget_shopping_cart_buttons', 'woocommerce_widget_shopping_cart_button_view_cart', 10 );`
But nothing happens, it look like the template file mini-cart.php isn't used at all. Is there another I have to look?

**Comment by crftwrk** [comment]
If you just want to remove the View cart button, simply download and install latest main branch. Cart buttons have been refactored in https://github.com/bootscore/bootscore/pull/755 and is already merged.

**Comment by finalwebsites** [comment]
Hi,
After I cloned the code from Github, I got the new version of wc-functions.php.
Thank you.
To hide the "View cart" button I used this filter:
`add_filter('bootscore/skip_cart', function() { return true; });`
And of course you need to empty the cart because the whole mini cart is stored in the browser

**Reply by crftwrk** [reply]
> To hide the "View cart" button I used this filter:

No need to add the filter because this is the default setting:

https://github.com/bootscore/bootscore/blob/eae3a62cb3f81d577c3685a5c43fa56cd7655f62/woocommerce/wc-functions.php#L58-L65
